From 6aeb0c5366a32f571de7e7480ae95f5fd9b394f1 Mon Sep 17 00:00:00 2001
From: "Tobias C. Berner" <tcberner@gmail.com>
Date: Wed, 10 May 2017 21:30:40 +0200
Subject: [PATCH] Fix filesytem type detection on FreeBSD

FreeBSD's df also uses -T to print the filesystem type:
>      -T      Include file system type.
---
 src/disklist.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/disklist.h b/src/disklist.h
index dd17a68..6e9845e 100644
--- src/disklist.h
+++ src/disklist.h
@@ -33,7 +33,7 @@
 
 static QLatin1String DF_Command = QLatin1String( "df" );
 
-#if defined(Q_OS_LINUX)
+#if defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD)
     static QLatin1String DF_Args = QLatin1String( "-kT" );
     static const bool No_FS_Type = false;
 #else
-- 
2.12.1

